packages/superset-ui-preset-chart-xy/src/Line/buildQuery.ts

```diff
@@ -2,5 +2,13 @@ import { buildQueryContext } from '@superset-ui/chart';
 import ChartFormData from './ChartFormData';
 
 export default function buildQuery(formData: ChartFormData) {
-  return buildQueryContext(formData);
+  const queryContext = buildQueryContext(formData);
+
+  // Enforce time-series mode
+  queryContext.queries.forEach(query => {
+    const q = query;
```

Comment:
nit: Should we make a copy? (I don't think it is necessary imo).

Reply:
umm good point. but since nobody has access to the original one I think it is ok to modify instead of cloning the entire `queryContext`.